### PR TITLE
registration form validations

### PR DESCRIPTION
**Pull request by olessiap**

### CODE REVIEW

File: website/src/pages/Register/RegisterForm.js [hunk 1]
```diff
@@ -117,6 +135,16 @@ export default function RegisterForm() {
       )
     case 2:
       return (
+<<<<<<< HEAD
```
Comment (nikhil-g777) [inline website/src/pages/Register/RegisterForm.js:138]: There seem to be some merge conflicts, can you resolve these

File: website/src/pages/Register/RegisterMain.js [hunk 2]
```diff
@@ -67,6 +78,24 @@ const RegisterMain = props => {
...
+
+  const handleInput = (e) => {
+    setEmptyFieldError(false)
+    let { value, name } = e.target;
+    setState((prevState) => ({
+      ...prevState,
+      [name]:value
+    }))
+  }
+
```
Comment (nikhil-g777) [inline website/src/pages/Register/RegisterMain.js:98]: In this page ( Step 1 ), we need all the fields to be required, I am able to click continue without filling any of the fields

File: website/src/pages/Register/RegisterStep2.js [hunk 3]
```diff
@@ -137,34 +155,41 @@ const RegisterStep2 = props => {
...
-            name="endYear"
-            onChange={props.handleInput}
-            input={<BootstrapInput />}
-          >
-          {years.map((year) => (
-            <MenuItem value={year}>{year}</MenuItem>
-          ))}
-          </Select>
-        </div>
+        {currentRole === false && (
```
Comment (nikhil-g777) [inline website/src/pages/Register/RegisterStep2.js:158]: Not sure what the issue is but I did fill all the fields for work experience but clicking continue doesn't take me to the next page
Screen Shot 2020-12-04 at 1 04 51 AM

**Review by nikhil-g777 — CHANGES_REQUESTED** [review]
Please have a look at the comments

File: website/src/pages/Register/RegisterForm.js (hunk 1 above)
Comment (olessiap) [reply website/src/pages/Register/RegisterForm.js]: heyo, i fixed the merge conflicts, sorry about that!

**Comment by olessiap** [comment]
hey hey, form validations should be working now! Let me know if I missed anything. Still need to do fix some of the theme styling @nikhil-g777

**Comment by nikhil-g777** [comment]
Hey, it looks good for the most part, just a couple of things we might have missed out. 
Can you have a look at this sheet that we spoke about and implement accordingly : https://docs.google.com/document/d/1wxQ4fmbwab5kSOhh5vYQjzjhz7qtqHTYDW0zFAH8mp4

It's alright if it takes longer, lets try doing these and other changes if you have any suggestions

**Review by nikhil-g777 — APPROVED** [review]

**Review by nikhil-g777 — APPROVED** [review]

**Pull request merged**